src/MetaCompilation.Analyzers/Core/xlf/MetaCompilationResources.cs.xlf

```diff
@@ -5,15 +5,15 @@
       <trans-unit id="AnalyzerDescription">
         <source>Type names should be all uppercase.</source>
         <target state="new">Type names should be all uppercase.</target>
-        <note>An optional longer localizable description of the diagnostic.</note>An optional longer localizable description of the diagnostic.</trans-unit>
+        <note />An optional longer localizable description of the diagnostic.</trans-unit>
```

Comment:
Your output looks like what v51 generated on first generation and subsequent update using v51 would produce the note both inside and outside the tag as before the diff. I suspect v51 was still used here somehow after deletion.